screens/Overview.js

```diff
@@ -0,0 +1,38 @@
+import { NavigationContainer } from '@react-navigation/native';
+import React from 'react';
+import { Text, Button } from 'react-native';
+import styled from 'styled-components/native';
+
+const OverviewContainer = styled.View`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`;
+
+const TextContainer = styled.Text`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 50%;
+  text-align: justify;
+`
+
+const Overview = ( {navigation} ) => {
+    const pressHandler = ( ) => {
+        navigation.goBack()
+    }
+
+    return (
+        <OverviewContainer>
+            <TextContainer>
+              So, how does that nested Stack-Navigation feel {navigation.getParam("name")}?
+              You can now either go back by using the button or the arrow.
+            </TextContainer>
+            <Button title="Go back" onPress={pressHandler}/>
```

Comment:
Your buttons are in the wrong position 😭 
![image](https://user-images.githubusercontent.com/50926159/115219927-4424ba00-a108-11eb-8790-7b2614b49638.png)